tasks/main.yml

```diff
@@ -6,6 +6,7 @@
   when: ansible_os_family == 'Debian'
 
 - name: Install Puppet.
+  become: true
```

Comment:
I assume my roles are run with become applied at the role or playbook level, so I don't add become to individual tasks in the roles.